spec/features/users/user_signs_up_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
+
+feature 'user registers', %Q{
+  As a visitor
+  I want to register
+  So that I can create an account
+} do
+
+  # Acceptance Criteria:
+  # * I must specify a valid email address,
+  #   password, and password confirmation
+  # * If I don't specify the required information, I am presented with
+  #   an error message
+
+  scenario 'provide valid registration information as regular user' do
+    visit new_user_registration_path
+
+    fill_in 'Email', with: 'john@example.com'
```

Comment:
Not wise to key in lots of strings here. Better to create an object and call on its attributes.